kpet/data.py

```diff
@@ -451,6 +474,56 @@ def convert_tree_arches(self):
 
             self.trees[name]["arches"] = list(tree_arches)
 
+    def convert_sets(self):
+        """ Convert each suite / cases sets from a list of regexes
+        to a list of set names matching those regexes
+
+        Returns:
+            Raises a schema.Invalid exception when finding an invalid regex
```

Comment:
To match the rest of the code (and e.g. Google guidelines) this should be written like this:
```
Raises:
    A schema.Invalid exception when finding an invalid regex.
```